samples/android/GleanCrashExample/build.gradle

```diff
@@ -0,0 +1,30 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext.kotlin_version = '1.3.50'
```

Comment:
This can be removed. Let's be consistent with the rest of the repo.

Reply:
See 'standalone' comment above